<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Snake</class>
 <widget class="QMainWindow" name="Snake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Snake</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="modeMenu">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="levelslist">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>131</y>
      <width>260</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>260</width>
      <height>180</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="btn_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_update">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="colLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
